src/shapes/textbox.class.ts

```diff
@@ -93,7 +110,7 @@ export class Textbox extends IText {
         charCount++;
         realLineCount++;
       } else if (
-        !this.splitByGrapheme &&
+        !this.resolveTextOverflowStrategy() &&
```

Comment:
guessing this...
no tests and not enough context to understand this.
Why doesn't it use the split lines?